ChangeLog

```diff
@@ -13,6 +13,8 @@ Default behavior changes
      Applied Sciences).
 
 Security
+   * Fix buffer overread vulnerability in ssl_parse_server_psk_hint
+     function.
```

Comment:
Please add the new entry at the end. We always sort these lists in chronological (merge) order.